### DISCUSSION TITLE
gephi input network correlation file

### DISCUSSION THREAD

**Discussion by bioinfonext** [gephi/gephi#2693] [Q&A]
Hello,

Could you please suggest me how I can import the below network in Gephi? In which file format should I save this data file?

```
Source	target	Cor	         p-value
G15	G2	0.852631579	3.53981E-06
G2	G15	0.852631579	3.53981E-06
G37	G12	0.783770109	7.1761E-05
G12	G37	0.783770109	7.1761E-05
G23	G4	0.775778925	9.48541E-05
G4	G23	0.775778925	9.48541E-05
G23	G2	0.774901347	9.7739E-05
G2	G23	0.774901347	9.7739E-05
G43	G30	0.749122807	0.000223207
G30	G43	0.749122807	0.000223207
G36	G28	0.747409319	0.000234991
```

Many thanks

**Comment by eduramiba** [comment]
This can be formatted as CSV/Excel and import it as edge table.

**Comment by bioinfonext** [comment]
Thank you so much. I was able to import data in Gephi. Could you please also help me -how I can colour edge attributes based on positive and negative correlation with different colors in Network?

Many thanks
bioinfonext
